guides/doc-Planning_for_Project/master.adoc

```diff
@@ -4,6 +4,12 @@ include::common/header.adoc[]
 
 = {PlanningDocTitle}
 
+// This guide is Katello specific, in particular the diagrams
+ifdef::foreman-el,foreman-deb[]
```

Comment:
So "HideDocument" here? Is there a diff between this and other guides that we're not even want to show it for nightly?

Reply:
No, HideDocument only hides it for stable branches but this whole document is not ready. Not even in nightly. Should the `HideDocument` attribute reflect that?

Reply:
Thinking about it: Maybe we should use "if not foreman-deb ..." for **all** instances where we want to hide a guide for a specific build target, irrespective of the version & rename "HideDocument" to something like "hide on stable branches", "not ready for production" or "nightly only". I like the latter.

Reply:
I've chosen to name it `HideDocumentOnStable`. The downside of nightly only is that the logic gets a bit complicated. Katello guides don't use it and always show. A good example where naming is hard.